docs/cli.md

```diff
@@ -1599,6 +1599,8 @@ If you want to grab cookies from another browser or want
 to directly pass the cookie as a string, check out the
 -c/--cookie flag.
 
+The profile_pic_url has a limited life time.
```

Comment:
Try something clearer like `Beware, instagram only provides temporary links, not permalinks, for profile picture urls retrieved as the "profile_pic_url" in the result. Be sure to download them fast if you need them.`